tests/jest/unit/model/StudyRequestBulk.spec.js

```diff
@@ -21,6 +21,7 @@ test('StudyRequest', () => {
     reasonOther: null,
     ccEmails: [],
     studyType: StudyType.TMC,
+    studyTypeOther: null,
```

Comment:
let's use `generateStudyRequest()`, `generateStudyRequestBulk()` here as well (and in `StudyRequest.spec.js`)